internal/connect/client.go

```diff
@@ -346,6 +350,13 @@ func DeactivateProduct(product Product) (Service, error) {
 
 // DeregisterSystem deletes current system in SMT/SCC
 func DeregisterSystem() error {
+	// In case fetching credentials fails do not break
+	// but continue to allow deregistration with remote API
+	creds, err := getCredentials()
+	if err == nil {
```

Comment:
This call should go in `Deregister` function, `deregisterSystem()` is also called early.

Reply:
thanks for finding the spot, deregistering was broken as you found out. I will move it there

Reply:
moved!